h/api/search/test/core_test.py

```diff
@@ -0,0 +1,30 @@
+import mock
+import pytest
+
+from h.api.search import core
+
+
+search_fixtures = pytest.mark.usefixtures('query', 'models')
+
+
+@search_fixtures
+def test_search_passes_private_to_AuthFilter(query):
+    request = mock.Mock()
+
+    core.search(request, mock.Mock(), private=True)
+
+    query.AuthFilter.assert_called_once_with(request, True)
```

Comment:
Could this be written as `assert_called_once_with(request, private=True)`? If so, that would be clearer.